src/api/app/views/webui2/webui/session/sso_confirm.html.haml

```diff
@@ -0,0 +1,17 @@
+- @pagetitle = 'First Login'
+
+.card
+  .card-body#loginform
+    .col-lg-6.pl-0
+      - if can_register
```

Comment:
While deploying this today on a new instance, we found out we actually want to disallow new registrations while allow new users log in with SSO.
The code doesn’t currently distinguish between these two different sign-ups (non-verified self-registration and a new user being created after confirmed by the SSO), so we ended up removing this condition here so that SSO can still allow new users in `deny` mode.
Ultimately it needs to be a separate setting.